client/state/partner-portal/licenses/test/hooks.js

```diff
@@ -26,7 +26,7 @@ import useTOSConsentMutation from 'calypso/state/partner-portal/licenses/hooks/u
 
 jest.mock( 'react-redux', () => ( {
 	useDispatch: jest.fn( () => null ),
-	useSelector: () => 1,
+	useSelector: jest.fn(),
```

Comment:
🤔 Any implications to this change? It's a common hook.

Reply:
Maybe this just to be extra careful:

```suggestion
	useSelector: jest.fn( () => 1 ),
```

Reply:
I am not sure what it does... it was a Yashwin suggestion